library/src/main/java/io/andref/rx/widgets/ListViewCardAdapter.java

```diff
@@ -0,0 +1,264 @@
+
+import static android.view.View.GONE;
+import static io.andref.rx.widgets.ListViewCard.TAG;
+
+public class ListViewCardAdapter extends RecyclerView.Adapter
+{
+    private List<ListViewCard.Item> mItems = new ArrayList<>();
+
+    private PublishSubject<Pair<ListViewCard.Item, Integer>> mItemClicks = PublishSubject.create();
+    private PublishSubject<Pair<ListViewCard.Item, Integer>> mIconClicks = PublishSubject.create();
+
+    private float mCellHeight;
+    private float mAvatarAlpha;
+    private float mIconAlpha;
+    private int mAvatarTint;
+
+    private boolean mShowLastDivider = true;
+
+    public ListViewCardAdapter(float cellHeight, float avatarAlpha, float iconAlpha, int avatarTint)
+    {
+        mCellHeight = cellHeight;
+        mAvatarAlpha = avatarAlpha;
+        mIconAlpha = iconAlpha;
+        mAvatarTint = avatarTint;
+    }
+
+    @Override
+    public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType)
+    {
+        switch (viewType)
+        {
+            default:
+                View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.rxw_avatar_with_two_lines_and_icon, parent, false);
+                final ViewHolder viewHolder = new ViewHolder(view, mCellHeight);
+
+                RxView.clicks(view)
+                        .takeUntil(RxView.detaches(parent))
+                        .subscribe(new Action1<Void>()
+                        {
+                            @Override
+                            public void call(Void aVoid)
+                            {
+                                mItemClicks.onNext(new Pair<>(mItems.get(viewHolder.getAdapterPosition()), viewHolder.getAdapterPosition()));
+                            }
+                        });
+
+                View iconView = view.findViewById(R.id.image_button);
+                RxView.clicks(iconView)
+                        .takeUntil(RxView.detaches(parent))
+                        .subscribe(new Action1<Void>()
+                        {
+                            @Override
+                            public void call(Void aVoid)
+                            {
+                                mIconClicks.onNext(new Pair<>(mItems.get(viewHolder.getAdapterPosition()), viewHolder.getAdapterPosition()));
+                            }
+                        });
+
+                return viewHolder;
+        }
+    }
+
+    @Override
+    public void onBindViewHolder(RecyclerView.ViewHolder holder, int adapterPosition)
+    {
+        switch (getItemViewType(adapterPosition))
+        {
+            default:
+                final ViewHolder viewHolder = (ViewHolder) holder;
+                viewHolder.bindItem(mItems.get(adapterPosition), mAvatarAlpha, mAvatarTint, mIconAlpha);
+                if (adapterPosition == getItemCount() - 1)
+                {
+                    if (mShowLastDivider)
+                    {
+                        viewHolder.showDivider();
```

Comment:
I know we keep wavering on what to call these methods, but I rather follow typical Android view style in this case. So `setDividerVisibility(boolean)`.